src/main/java/oshi/software/os/mac/local/CentralProcessor.java

```diff
@@ -32,8 +37,14 @@
 	 * @return Processor vendor.
 	 */
 	public String getVendor() {
-		if (_vendor==null)
-			_vendor=ExecutingCommand.getFirstAnswer("sysctl -n machdep.cpu.vendor");
+		if (_vendor == null) {
+			int[] mib = { SystemB.CTL_MACHDEP, SystemB.MACHDEP_CPU,
+					SystemB.MACHDEP_CPU_VENDOR };
+			IntByReference size = new IntByReference(100);

```

Comment:
Is this a documented max constant? I am not familiar with the Mac API, Is there a way to do the call in 2 phases where you get the size first, alloc memory then do it again?